### ISSUE TITLE
v4 - It seems that some of templates for scaffolding need to be updated (tested in PHP 7.4)

### ISSUE THREAD

**Issue by stmkjp** [phalcon/phalcon-devtools#1411]
Hi, 
thanks for all the efforts on the product!
I'm new to phalcon and trying to install & follow tutorials, so please understand if the issues described below are nothing but my ignorance.
### Expected and Actual Behavior

> **Describe what you are trying to achieve and what goes wrong.**

When scaffolds are generated, search page produces errors:

1.  **template for generating controller file needs to be updated**
 `Fatal error: Uncaught Error: Call to undefined method Phalcon\Paginator\Adapter\Model::getPaginate() ...`
 ==> According to the v4 changelog, getPaginate() is replaced by paginate(), so
        I guess `$paginator->getPaginate()`should be changed to `$paginator->paginate()`

==> But it also produces an notice & warning saying
`Notice: Undefined index: model in ...
Warning: call_user_func() expects parameter 1 to be a valid callback, first array member is not a valid class name or object in ...` 

==> So, it seems that 
```
$users = Users::find($parameters);
        if (count($users) == 0) {
            $this->flash->notice("The search did not find any users");

            $this->dispatcher->forward([
                "controller" => "users",
                "action" => "index"
            ]);

            return;
        }

        $paginator = new Paginator([
            'data' => $users,
            'limit'=> 10,
            'page' => $numberPage,
        ]);
```

section also needs to be updated, something like this:

```
$paginator = new Paginator([
	    	'model'=>Users::class,
			 'parameters'=>$parameters,
			 'limit'=>10,
			  'page'=>$numberPage

		    ]
	    );
        if ($paginator->paginate()->getTotalItems() == 0) {
            $this->flash->notice("The search did not find any users");

            $this->dispatcher->forward([
                "controller" => "users",
                "action" => "index"
            ]);

            return;
        }
```
**but I'm not sure if this is the right fix or I'm doing something wrong.**

2. **generated view file also needs to be updated (i guess)**
    received $page->items look like a plain array, not object.
   so, 

```
                <td><?php echo $catalog->id ?></td>
               <td><?php echo $catalog->somefiled ?></td>
            ...
           <td><?php echo $this->tag->linkTo(["catalogs/edit/" . $catalog->getId(), "Edit"]); ?></td>

           ...
<div class="row">
    <div class="col-sm-1">
        <p class="pagination" style="line-height: 1.42857;padding: 6px 12px;">
            <?php echo $page->current, "/", $page->total_pages ?>
        </p>
    </div>
    <div class="col-sm-11">
        <nav>
            <ul class="pagination">
                <li><?php echo $this->tag->linkTo(["catalogs/search", "First", 'class' => 'page-link']) ?></li>
                <li><?php echo $this->tag->linkTo(["catalogs/search?page=" . $page->before, "Previous", 'class' => 'page-link']) ?></li>
                <li><?php echo $this->tag->linkTo(["catalogs/search?page=" . $page->next, "Next", 'class' => 'page-link']) ?></li>
                <li><?php echo $this->tag->linkTo(["catalogs/search?page=" . $page->last, "Last", 'class' => 'page-link']) ?></li>
            </ul>
        </nav>
    </div>
</div>

```
 need to be :
```
                <td><?php echo $catalog['id] ?></td>
               <td><?php echo $catalog['somefiled'] ?></td>
            ...
           <td><?php echo $this->tag->linkTo(["catalogs/edit/" . $catalog['id']>getId(), "Edit"]); ?></td>
...

<div class="row">
    <div class="col-sm-1">
        <p class="pagination" style="line-height: 1.42857;padding: 6px 12px;">
            <?php echo $page->current, "/", $page->last ?>
        </p>
    </div>
    <div class="col-sm-11">
        <nav>
            <ul class="pagination">
                <li><?php echo $this->tag->linkTo(["catalogs/search", "First", 'class' => 'page-link']) ?></li>
                <li><?php echo $this->tag->linkTo(["catalogs/search?page=" . $page->previous, "Previous", 'class' => 'page-link']) ?></li>
                <li><?php echo $this->tag->linkTo(["catalogs/search?page=" . $page->next, "Next", 'class' => 'page-link']) ?></li>
                <li><?php echo $this->tag->linkTo(["catalogs/search?page=" . $page->last, "Last", 'class' => 'page-link']) ?></li>
            </ul>
        </nav>
    </div>
</div>

```
Also, I'm not sure if there's another way or...

### Details

* System info and versions (if possible): (`phalcon info`)
* Phalcon Framework version: (`php --ri phalcon`) v4.0.0
* PHP Version: (`php -v`) 7.4
* Operating System: MacOS Catalina
* Server: Nginx 
* Other related info (Database, table schema): 
DB: mariadb(mysql)
phalcon devtool: v4.0.0

**Comment by Jeckerson** [comment]
Related - #1420

**Comment by jenovateurs** [comment]
Fixed in #1436
